cfme/tests/configure/test_workers.py

```diff
@@ -27,3 +54,42 @@ def test_restart_workers(appliance):
     wait_for(lambda: len(pids) == len(worker.get_all_worker_pids()),
              fail_func=worker.parent.reload_workers_page, num_sec=1800, delay=10,
              message="Wait for all original workers are back online")
+
+
+@pytest.mark.tier(0)
+@pytest.mark.parametrize("dropdown", DROPDOWNS, ids=IDS)
```

Comment:
Consider using a namedTuple for the pairing, to keep them in order should we get new worker types. Explicitly parametrize here against `[t.dropdown for t in DROPDOWN_TUPLES]` and pass `ids=[t.id for t in DROPDOWN_TUPLES]`

Reply:
I always wondered if there was an easy way to do something like this in python. I will update this to use a namedtuple. :+1: